api/server/router/image/image_routes.go

```diff
@@ -220,7 +221,7 @@ func (s *imageRouter) deleteImages(ctx context.Context, w http.ResponseWriter, r
 	name := vars["name"]
 
 	if strings.TrimSpace(name) == "" {
-		return fmt.Errorf("image name cannot be blank")
+		return errors.New("image name cannot be blank")
```

Comment:
Looks like parsing the reference also returns an error for this case. If this check is disabled;

```
$ docker image rm ""
Error response from daemon: Error parsing reference: "" is not a valid repository/tag: repository name must have at least one component
```

Guess "image name cannot be blank" is a bit more friendly